osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -172,22 +179,16 @@ protected override void OnHoverLost(HoverLostEvent e)
         {
             if (GetContainingInputManager()?.HoveredDrawables.Contains(VisualSettings) == true)
             {
-                // Update background elements is only being called here because blur logic still exists in Player.
```

Comment:
whyyyyyyyyyy is this gone??????
it's a very important comment.

Reply:
update background elements isn't being called anymore though, so its no longer correct.
Also the line afterwards explicitly mentions the issue this resolves so... not sure what you mean

Reply:
a comment should be added back to explain why the onhover/hoverlost is applied to the screen and not the visual settings overlay.

Reply:
Added new comments regarding why this hover logic exists